asciimatics/widgets/button.py

```diff
@@ -17,16 +17,16 @@ class Button(Widget):
 
     __slots__ = ["_text", "_add_box", "_on_click", "_label"]
 
-    def __init__(self, text, on_click, label=None, add_box=True, **kwargs):
+    def __init__(self, text, on_click, label=None, add_box=True, name=None, **kwargs):
         """
         :param text: The text for the button.
         :param on_click: The function to invoke when the button is clicked.
         :param label: An optional label for the widget.
```

Comment:
Looks good.  But can you please add the docstring for the new parameter.  Feel free to borrow the text from other widgets to keep it quick n easy.

Reply:
Sure thing.  Added a docstring for add_box while I was there.

Reply:
Nice!  Thanks